README.md

```diff
@@ -348,13 +348,15 @@ Then, edit the configuration file. Only thing that is necessary to change is `gh
 
 Depending on your browser preference you might want to change `browser`, this is the command that will be invoked when clicking on an issue number. It defaults to `null` which indicates to use the system web browser.
 
-If you want to see PR status (and other issue details like labels), point `meta` for the `bitcoin/bitcoin` repository to an up-to-date checkout of [bitcoin-gh-meta](https://github.com/zw/bitcoin-gh-meta). One way to keep this repository up to date is with a crontab entry that periodically does `git pull`.
+If you want to see PR status (and other issue details like labels), point `meta` for the `bitcoin/bitcoin` repository to an up-to-date checkout of [bitcoin-gh-meta](https://github.com/zw/bitcoin-gh-meta).
 ```
     "meta": {
         "bitcoin/bitcoin": "/path/to/bitcoin-gh-meta"
     },
 ```
 
+To keep this repository up to date you can set the interval in seconds in 'auto-update', default is 0 (i.e. no auto-update).
```

Comment:
naming nit, perhaps use snakecase like `label_prio` for `auto_update`
```suggestion
To keep this repository up to date you can set the interval in seconds in 'auto_update', default is 0 (i.e. no auto-update).
```

Reply:
It looks like bitcoin-gh-meta performs a refresh every 2 hours. Maybe mention that in the `auto_update` documentation. After the first auto-update git pull, settings under 2 hours won't provide any benefit.

Perhaps a refresh could be done on launch if auto_update > 0.

Reply:
Done, and done.. thanks for the suggestions!!